presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -474,8 +474,8 @@ public PlanWithProperties visitSort(SortNode node, Context context)
                 // skip the SortNode if the local properties guarantee ordering on Sort keys
                 // TODO: This should be extracted as a separate optimizer once the planner is able to reason about the ordering of each operator
                 List<LocalProperty<Symbol>> desiredProperties = new ArrayList<>();
-                for (Symbol symbol : node.getOrderBy()) {
-                    desiredProperties.add(new SortingProperty<>(symbol, node.getOrderings().get(symbol)));
+                for (Symbol symbol : node.getOrderingScheme().getOrderBy()) {
+                    desiredProperties.add(new SortingProperty<>(symbol, node.getOrderingScheme().getOrdering(symbol)));
```

Comment:
either commits are not sorted or this commit does not compile as there is no `OrderingScheme#getOrdering` method yet